### DISCUSSION TITLE
Rotating one third, a half and two thirds with Ctr+Opt+arrow not working any more

### DISCUSSION THREAD

**Discussion by shinokada** [rxhanson/Rectangle#686] [General]
macOS version: v12.1
Rectangle version: v0.49(54)

It still works on one of my laptops using Ctr+Opt+arrow to toggle a third, a half, and two-thirds.
But on my iMac, it doesn't do it.

Is there any setting I have to set?

**Comment by rxhanson** [comment]
Change the "Repeated commands" setting in the settings tab of the prefs window to "cycle ... on half actions".

**Comment by shinokada** [comment]
Thanks. I got it.
